docs/docs/firmware/touch-panel-esp32.md

```diff
@@ -2071,6 +2071,45 @@ RGB color for a tile icon (defaults to white if the tile state is "off", or the
 [comment]: <> ([TODO] Screen Payloads explanation)
 These commands are specific to an individual screen.
 
+## Remove a screen from the configuration
+Removes a screen from the configuration.
+
+[comment]: <> (START of JSON Example)
+:::: code-group
+::: code-group-item Command
+```json
+{
+  "screens":[
+    {
+      "screen": 2,
+      "action": "remove"
+    }
+  ]
+}
+```
+
+### JSON parameters
+| Parameter            | Type     | Options    | Description                       |                                                               |
+|:---                  |:---:     |:---:       |:---                               |:---                                                           |
+| `screen`             | *Number* | n/a        | Screen number to be removed       | <Badge type="warning" text="Required" vertical="bottom" /> |  
+| `action`             | *String* | `"remove"` | Command to remove screen          | <Badge type="warning" text="Required" vertical="bottom" />    |
+
+<Badge type="warning" text="MQTT Topic" vertical="middle" />
+
+```cmnd/<device-client-id>```
+:::
+::::
+[comment]: <> (END of JSON Example)
+
+::: tip
+
+Removing a screen removes all tiles on this screen as well.
+
+Removing sreen 1 (Home Screen) technically removes screen 1 and creates an empty screen 1 (Home Screen)

```

Comment:
Typo - `sreen`

Reply:
Your proposal sounds good, but screen romoval has a different payload than tile. 
The tile actions can be put together, but I don't see how that would look together with the sceen. Any thoughts?

Reply:
Yeah I was thinking a section for screen actions and another for tile actions - each with a list of supported `action` commands.

Reply:
Ok, I'll give it a try

Reply:
Yep - so I would see these `Actions` being under those existing screen/tile sections.

Reply:
Personally I would just do what Moin originally had as its a feature at tile or screen level. Thus document the payloads against each with example payloads as they differ like everything else.

We are adding another thing now called "Actions" that needs explaining! For example If i was looking for an ability to disable a tile id look in the tile payloads section, Just my opinion.

Reply:
so screen removal stays where it is (under `screen payloads`)
tile removal and disable will be combined to an `action topic` and move under` tile payloads`

Reply:
Oh ok maybe I've misunderstood lol

Reply:
I'm happy to go with whatever you guys think is best - just thought it made more sense to have all actions for each entity type in one place, rather than duplicating the `remove` and `disable` tile actions etc.

Reply:
Suppose it can always change if people are struggling, do what feels right to you Moin 👍